client/ayon_core/modules/deadline/plugins/publish/collect_deadline_server_from_instance.py

```diff
@@ -66,7 +91,7 @@ def _collect_deadline_url(self, render_instance):
 
         default_servers = {
             url_item["name"]: url_item["value"]
-            for url_item in deadline_settings["deadline_urls"]
+            for url_item in deadline_settings["deadline_server_info"]
         }
```

Comment:
Is there a reason we're not filtering the `deadline_server_info` already to whatever is a valid project server? So that we filter it once in a context plug-in as opposed to per instance here?

Reply:
Will go away in some next PR.